library/EasyPipeline/src/Easy.hs

```diff
@@ -36,59 +39,77 @@ import qualified Juvix.Frontend.Types.Base as Frontend
 import qualified Juvix.FrontendDesugar as FrontDesugar
 import Juvix.Library
 import qualified Juvix.Library.Feedback as Feedback
+import qualified Juvix.Library.HashMap as Map
 import qualified Juvix.Library.NameSymbol as NameSymb
 import qualified Juvix.Library.Sexp as Sexp
 import qualified Juvix.Pipeline as Pipeline
 import qualified Juvix.Pipeline.Compile as Compile
+import qualified Juvix.ToCore.Types as ToCore.Types
 import qualified Text.Pretty.Simple as Pretty
 import Prelude (error)
+import qualified Prelude (Show (..))
```

Comment:
Separate from this PR: shouldn't standardlibrary include this?

Reply:
Protolude, has it's own version of `Show` and so if you make this standard, it'd be ambiguities all over the place

Reply:
Ah OK.